packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -17,3 +17,18 @@ export const correctStackForCrossDomainError = (serializedError: any, userInvoca
 
   return reifiedError
 }
+
+export const serializeRunnable = (runnable) => {
+  if (!runnable) return undefined
+
+  const fields = _.pick(runnable, ['id', '_currentRetry', 'type', 'title', 'parent', 'ctx'])
```

Comment:
Right now I'm just picking the fields needed but we could serialize all of the fields that can be serialized. Thoughts?